content-src/styles/variables.scss

```diff
@@ -81,9 +81,13 @@ $search-input-left-label-width: 35px;
 $search-input-border-color: #BFBFBF;
 $search-blue: #0996F8;
 $search-shadow: rgba(96, 181, 255, 0.75);
+$search-box-shadow: rgba(0, 0, 0, 0.1);
 $search-button-grey: #FBFBFB;
 $search-button-border-color: #D4D4D4;
 $search-button-width: 64px;
+$search-text: #666;
+$search-section-background: #F7F7F7;
+$search-border-color: #EAEAEA;
```

Comment:
Not sure if we want to use `$very-light-grey: #EBEBEB;` here since the colors are remarkably similar.
Maybe we should run something like [**colorguard**](https://www.npmjs.com/package/colorguard) to make sure we aren't using a bunch of excessive colors that are super close to other colors in the style guide.